examples/Storage/Storage.Tests.ps1

```diff
@@ -1,78 +1,138 @@
 BeforeAll {
   Import-Module Az.InfrastructureTesting
+
+  $Script:rgName = 'rg-test'
+  $Script:accountName = 'azbenchpressstorage'
+  $Script:location = 'westus3'
 }
 
 Describe 'Verify Storage Account' {
-  it 'Should contain a Storage Account with the given name' {
+  BeforeAll {
+    $Script:noAccountName = 'noazbenchpressstorage'
+  }
+
+  It 'Should contain a Storage Account with the given name - Confirm-AzBPResource' {
     #arrange
-    $rgName = 'rg-test'
-    $name = 'azbenchpressstorage'
+    $params = @{
+      ResourceType = "StorageAccount"
+      ResourceGroupName = $rgName
+      ResourceName = $accountName
+    }
 
     #act
-    $result = Confirm-AzBPStorageAccount -ResourceGroupName $rgName -Name $name
+    $result = Confirm-AzBPResource @params
 
     #assert
     $result.Success | Should -Be $true
   }
 
-  it 'Should not contain a Storage Account with the given name' {
+  It "Should contain a Storage Account named $accountName - ConfirmAzBPResource" {
     #arrange
-    $rgName = 'rg-test'
-    $serverName = 'noazbenchpressstorage'
+    $params = @{
+      ResourceType = "StorageAccount"
+      ResourceGroupName = $rgName
+      ResourceName = $accountName
+      PropertyKey = 'StorageAccountName'
+      PropertyValue = $accountName
+    }
+
+    #act
+    $result = Confirm-AzBPResource @params
+
+    #assert
+    $result.Success | Should -Be $true
+  }
+
+  It 'Should contain a Storage Account with the given name' {
+    #act
+    $result = Confirm-AzBPStorageAccount -ResourceGroupName $rgName -Name $accountName
+
+    #assert
+    $result.Success | Should -Be $true
+  }
+
+  It 'Should not contain a Storage Account with the given name' {
 
     #act
     # The '-ErrorAction SilentlyContinue' command suppresses all errors.
     # In this test, it will suppress the error message when a resource cannot be found.
     # Remove this field to see all errors.
-    $result = Confirm-AzBPStorageAccount -ResourceGroupName $rgName -Name $serverName -ErrorAction SilentlyContinue
+    $result = Confirm-AzBPStorageAccount -ResourceGroupName $rgName -Name $noAccountName -ErrorAction SilentlyContinue
 
     #assert
     $result.Success | Should -Be $false
   }
 
-  it 'Should contain a Storage Account named azbenchpressstorage' {
-    #arrange
-    $rgName = 'rg-test'
-    $name = 'azbenchpressstorage'
-
+  It 'Should contain a Storage Account named $accountName' {
     #act
-    $result = Confirm-AzBPStorageAccount -ResourceGroupName $rgName -Name $name
+    $result = Confirm-AzBPStorageAccount -ResourceGroupName $rgName -Name $accountName
 
     #assert
     $result | Should -BeDeployed
   }
 
-  it 'Should contain a Storage Account named azbenchpressstorage in westus3' {
+  It "Should contain a Storage Account named $accountName in $location" {
```

Comment:
Do you know if the variables show the correct naming when printing out/is this the correct format for vars in a string?